README.md

```diff
@@ -28,4 +31,53 @@ To build and deploy the toradex image, simply call the appropriate dobi resource
 ./dobi.sh deploy
 ```
 
-*Note: the seperated prepare and build/deploy steps are used to let the user decide which sources should be build, e.g. if the user has modifications those should be build. Another prepare step will resync will upstream repos.*
\ No newline at end of file
+*Note: the seperated prepare and build/deploy steps are used to let the user decide which sources should be build, e.g. if the user has modifications those should be build. Another prepare step will resync will upstream repos.*
+
+## CI/CD build and deploy
+
+### Prerequisites
+
+-   Concourse CI, MinIO and Harbor e.g. via local-dev-environment (<https://github.com/ci4rail/local-dev-environment>)
+-   docker (<https://docs.docker.com/install/>)
+
+### Usage
+
+To build and deploy the toradex image, you must set the pipeline to the concourse CI server.
+
+First download `fly` from your concoruse server. The following example uses a local concourse server that is reachable at https://localhost:9000.
+
+```bash
+sudo wget http://localhost:9090/api/v1/cli?arch=amd64&platform=linux -O /usr/local/bin/fly
+sudo chmod +x /usr/local/bin/
+```
+
+Then login to your concourse instance.
+
+```bash
+fly --target local login --concourse-url http://localhost:9090 -u <user> -p <password>
+```
+
+You can copy `ci/config.yaml` and `ci/credentials.template.yaml` and adapt it to your needs. 
```

Comment:
File is credentials.example, not credentials.template

Reply:
fixed